spec/system/casa_admins/edit_spec.rb

```diff
@@ -6,27 +6,53 @@
   before { sign_in admin }
 
   context "with valid data" do
-    it "can successfully edit user email and display name" do
-      expected_email = "root@casa.com"
+    it "can successfully edit user display name and phone number" do
       expected_display_name = "Root Admin"
       expected_phone_number = "+14398761234"
 
       visit edit_casa_admin_path(admin)
 
-      fill_in "Email", with: expected_email
       fill_in "Display name", with: expected_display_name
       fill_in "Phone number", with: expected_phone_number
 
       click_on "Submit"
 
       admin.reload
 
-      expect(page).to have_text "New admin created successfully"
-      expect(admin.email).to eq expected_email
+      expect(page).to have_text "New admin created successfully" # should be updated
+
       expect(admin.display_name).to eq expected_display_name
       expect(admin.phone_number).to eq expected_phone_number
     end
   end
+  context "with valid email data" do
+    before do
+      visit edit_casa_admin_path(admin)
+      @old_email = admin.email
+      fill_in "Email", with: "new_admin_email@example.com"
+
+      click_on "Submit"
+      admin.reload
+    end
+    it "sends a confirmation email upon submission and does not change the user's displayed email" do
```

Comment:
it looks like most of these other system specs have a space between each expectation and think that makes it more readable.

need to add spaces between the `end` of the previous spec and the `it` for the current one.

Reply:
Totally agree, for some reason I thought one of the linters would help with that. Added the spaces.